sci-mathematics/singular/singular-4.2.1-r1.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
If we are revbumping the ebuild anyway, can we please also bump to `EAPI=8`

Reply:
Done